app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>BudjetApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
+    <%# <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script> %>
```

Comment:
Overall well done with Budget application 🚀 
The design is simplistic and is aligned with the Behance design guildlines. 💯 

#### Layout Composition
The pages fulfill the layout guidelines and meet all the project requirements. 

Homepage | Transactions
------------- | -------------
![image](https://github.com/clin2on3mun/BudgetApp/assets/38088818/1cd28387-3367-433f-b382-36f78e06e392)  | ![image](https://github.com/clin2on3mun/BudgetApp/assets/38088818/b9756329-43fa-43d0-bcc4-964208ff0816)

#### Tests
Well done with all the unit and integration specs!!! 🚀 

![image](https://github.com/clin2on3mun/BudgetApp/assets/38088818/ec94712e-a108-4b04-b22a-64bc93de6170)